### DISCUSSION TITLE
Elasticalert stop working

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#1957] [Unsupported Versions]
Hi all, 

I noticed when I ran the so-feature-enable, my alerts stop coming through Kibana and I'm getting this message in the log file 

![image](https://user-images.githubusercontent.com/59517722/99404733-f9351b00-28b9-11eb-814f-2d8a9a2f4be9.png)

This is on our manager node. Can anyone help me?

**Comment by weslambert** [comment]
Are you able to to the following?

`curl http://localhost:9200` 
or 
`curl -k https://localhost:9200`
